_includes/contact.html

```diff
@@ -0,0 +1,40 @@
+<div class="line"></div>
+<div class="contact">
+  {% if person %}
+    {% if showname != false %}<span class="name">{{ person.fullname }}</span><br>{% endif %}
+    Email: <a href="mailto:{{ person.email }}">{{ person.email }}</a>
+    {% if person.linkedin %}LinkedIn: <a href="{{ person.linkedin }}">{{ person.fullname }}</a><br>{% endif %}
+    {% if person.phone %}Phone: <span class="phone">{{ person.phone }}</span>{% endif %}
+    {% if person.twitter %}Twitter: <a href="twitter.com/{{ person.twitter }}">@{{ person.twitter }}</a>{% endif %}
+  {% endif %}
+  <br><br>
+</div>
+
+<style>
+  .line {
+    background-color: #707070;
```

Comment:
Consistency.  Why do you do rgb(r, g, b) below, and then use a hex code here?

Reply:
I added the rgb because that was what popped up in Atom, I usually use hex codes, and didn't get around to changing the old ones.